src/transforms.rs

```diff
@@ -161,5 +163,11 @@ mod tests {
             merged_rec[2],
             "AB  - This is the abstract's first line and this is its second line; with conclusion."
         );
+
+        let csl = medline_to_csl(merged_rec.into_iter()).unwrap();
+
+        let reduced = reduce_authors(csl);
+        let names: Vec<&CSLValue> = reduced.iter().filter(|x| x.is_name()).collect();
+        assert_eq!(names.len(), 2);
```

Comment:
two elements, but can you test that they are what you expect?

I also imagine since we always take the first one listed, that results are dependent on input source order, which I'll create an issue for

Reply:
I will add a test. Yes, it should take the first author line it observes for each duplicate author. Not sure how we would decided the "correct" one. I haven't explored the `nbib` and `CSL` formats so I am not sure how that should be handled.

Reply:
addressed in f8246a0